pgjdbc/src/test/java/com/yugabyte/FallbackOptionsLBTest.java

```diff
@@ -24,8 +24,12 @@ public static void main(String[] args) throws SQLException, ClassNotFoundExcepti
     }
     Class.forName("org.postgresql.Driver");
 
-    checkBasicBehavior();
-    checkNodeDownBehavior();
+    //System.out.println("Running checkBasicBehavior() ....");
+    //checkBasicBehavior();
+    //System.out.println("Running checkNodeDownBehavior() ....");
+    //checkNodeDownBehavior();
+    System.out.println("Running checkNodeDownBehaviorMultiFallback() ....");
+    checkNodeDownBehaviorMultiFallback();
```

Comment:
Is this test case the same as one in `FallbackLBTestExtended`?

Reply:
No, in this test case we are closing all existing connection before any new node comes up or goes down, whereas for the test in `FallbackLBTestExtended` the connections are not closed.